turbopack/crates/turbopack-ecmascript/src/tree_shake/graph.rs

```diff
@@ -1073,8 +1079,13 @@ impl DepGraph {
                         exports.push((default_var.to_id(), "default".into()));
                     }
                     ModuleDecl::ExportDefaultExpr(export) => {
-                        let default_var =
-                            private_ident!(magic_identifier::mangle("default export"));
+                        // Mirror what `EsmModuleItem::code_generation` does, these are live
+                        // bindings if the class/function has an identifier.
+                        let default_var = Ident::new(
```

Comment:
nit: Ident has `new_no_ctxt` which uses `Default::default()` for the third argument.
Not sure if it worths changing, though